pkgs/games/kabeljau/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenvNoCC, lib, fetchFromGitea, bash, dialog, makeWrapper }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "kabeljau";
+  version = "1.0.1";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "papojari";
+    repo = "kabeljau";
+    rev = "v${version}";
+    sha256 = "sha256-LOvr5fgSUTXnYhbVmynCCjo0W098jKWQnFULtIprE3M=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  postPatch = ''
+    patchShebangs --host ${pname}
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp ${pname}.sh $out/bin/${pname}
+    wrapProgram $out/bin/${pname} --prefix PATH : ${
+      lib.makeBinPath [ bash dialog ]
```

```suggestion
    wrapProgram $out/bin/${pname} --suffix PATH : ${
      lib.makeBinPath [ bash dialog ]
```
Why do we add bash here? Is that the shebang? If so we don't need this.

---

You mean we don't need bash in the PATH? dialog is needed. I mean, that's not in the shebang.